app/src/vanilla/java/cm/aptoide/pt/app/view/AppCoinsInfoFragment.java

```diff
@@ -0,0 +1,213 @@
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+import cm.aptoide.analytics.implementation.navigation.ScreenTagHistory;
+import cm.aptoide.pt.R;
+import cm.aptoide.pt.utils.AptoideUtils;
+import cm.aptoide.pt.view.AppCoinsInfoPresenter;
+import cm.aptoide.pt.view.BackButtonFragment;
+import com.jakewharton.rxbinding.view.RxView;
+import javax.inject.Inject;
+import rx.Observable;
+import rx.subjects.PublishSubject;
+
+/**
+ * Created by D01 on 30/07/2018.
+ */
+
+public class AppCoinsInfoFragment extends BackButtonFragment implements AppCoinsInfoView {
+
+  public static final String APPCWALLETPACKAGENAME = "com.appcoins.wallet";
+  @Inject AppCoinsInfoPresenter appCoinsInfoPresenter;
+  private Toolbar toolbar;
+  private PublishSubject<Void> coinbaseClickSubject;
+  private View bdsCardView;
+  private TextView appcMessageAppcoinsSection2a;
+  private Button installButton;
+  private TextView appcMessageAppcoinsSection2b;
+  private ClickableSpan coinbaseClickListener;
+
+  @Override public void onCreate(@Nullable Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    getFragmentComponent(savedInstanceState).inject(this);
+    coinbaseClickSubject = PublishSubject.create();
+    coinbaseClickListener = new ClickableSpan() {
+      @Override public void onClick(View view) {
+        if (coinbaseClickSubject != null) {
+          coinbaseClickSubject.onNext(null);
+        }
+      }
+    };
+  }
+
+  @Override public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
+    super.onViewCreated(view, savedInstanceState);
+    toolbar = (Toolbar) view.findViewById(R.id.toolbar);
+    bdsCardView = view.findViewById(R.id.product_bdsWallet_cardview);
+    installButton = (Button) view.findViewById(R.id.appview_install_button);
+    appcMessageAppcoinsSection2a =
+        (TextView) view.findViewById(R.id.appc_message_appcoins_section_2a);
+    appcMessageAppcoinsSection2b =
+        (TextView) view.findViewById(R.id.appc_message_appcoins_section_2b);
+
+    TextView appcMessageAppcoinsSection3 =
+        (TextView) view.findViewById(R.id.appc_message_appcoins_section_3);
+    TextView appcMessageAppcoinsSection4 =
+        (TextView) view.findViewById(R.id.appc_message_appcoins_section_4);
+
+    setupTextView(getString(R.string.appc_short_get_appc),
+        getString(R.string.appc_message_appcoins_section_3), appcMessageAppcoinsSection3);
+    setupTextView(getString(R.string.appc_short_spend_appc),
+        getString(R.string.appc_message_appcoins_section_4), appcMessageAppcoinsSection4);
+
+    ((TextView) bdsCardView.findViewById(R.id.product_title_textview)).setText(
+        getString(R.string.appc_title_settings_appcoins_wallet));
+    ((ImageView) bdsCardView.findViewById(R.id.product_icon_imageview)).setImageDrawable(
+        ContextCompat.getDrawable(getContext(), R.drawable.appcoins_wallet_icon));
+
```

Comment:
I did that to make the card view feature independent, so that it can be used in later features without being associated directly to this ticket, but I guess maybe there's another way to do this